### ISSUE TITLE
SimpleRepository Fail to Map Property of type Boolean (or bool) to SQLite

### ISSUE THREAD

**Issue by cfdelaune** [subsonic/SubSonic-3.0#61]
When I create an object with a Property of type Boolean (or bool) and use SimpleRepository to map it to SQLite, the column gets successfully created, but upon retrieval of the data, the following error gets thrown:

"Object of type 'System.Byte' cannot be converted to type 'System.Boolean'."

I posted this question on StackOverFlow and got an answer that I think might mean this is a bug.  Here is my post:

http://stackoverflow.com/questions/1137649/simplerepository-boolean-mapping-problem

Basically, in the SQLiteSchema.cs file, in the GetNativeType(DbType dbType) method, it looks like in the case of DbType.Boolean, the type was being set as "tinyint".  I checked this and indeed, the column in my SQLite database was of type "tinyint".  I changed this to "boolean", recompiled SubSonic and that seems to have resolved the problem.

Thanks!
